sinon/index.d.ts

```diff
@@ -427,7 +427,7 @@ declare namespace Sinon {
 
     // Utility overridables
     interface SinonStatic {
-        createStubInstance(constructor: any): SinonStub;
+        createStubInstance(constructor: any): any;
```

Comment:
Once a TS2.1 version has been published we don't have a way to go backwards and publish a new TS2.0 version. But NPM will continue to store the old versions, and we don't update the `latest` package (the one NPM installs by default) until a month after a compiler release. See Microsoft/types-publisher#229.